rust/integration-tests/src/sdk_tests/events_processor_tests.rs

```diff
@@ -137,10 +156,7 @@ mod tests {
             .map(|txn| serde_json::from_slice(txn).expect("Failed to deserialize transaction"))
             .collect();
 
-        let starting_version = transaction_batches
-            .first()
-            .expect("No transactions found")
-            .version;
+        let starting_version = 1;
```

Comment:
yeah that makes sense, I also thought it would be a devEx gap. then, it would also makes more sense to name the folder with original version for us too, when the custom file name isn't specified.

Reply:
one downside of stubbing out txn_versions would confuse users when they write their validation logic. we would have to tell them to use version 1, 2, 3, to query db.

Reply:
hmm what if instead of stubbing, the mock grpc just returns the transactions sorted in order of version?

Reply:
so then user uses [txn3, txn1, txn5] as an input, then grpc retruns [txn1, txn3, txn5]?

Reply:
Yeah exactly. I guess we have to choose between 
1. whether mock grpc allows gaps in the transaction batches 
2. stubbing out transaction versions -> you lose contextual information about the transaction 
3. require that scenario test txn's have no gaps (this is hard since there's no guarantee when submitting txn's that there won't be a gap)

Taking a step back, I think the goal here is to be able to test multiple txn's in a single test, so I think #1 could work here. The risk I see with #1 is if the user is writing a multi-txn test and their processor panics if it sees gaps within a txn batch. I think this is a minimal risk; the user could fallback to using single-txn test case.

Reply:
yeah the goal of this test is to processors not to test grpc, so it makes sense to allow mock grpc with gaps.